modules/deployment/src/test/java/org/mule/runtime/module/deployment/logging/LoggingSeparationTestCase.java

```diff
@@ -6,24 +6,32 @@
  */
 package org.mule.runtime.module.deployment.logging;
 
+import static org.mule.test.allure.AllureConstants.ComponentsFeature.LoggerStory.LOGGER;
+import static org.mule.test.allure.AllureConstants.IntegrationTestsFeature.INTEGRATIONS_TESTS;
+
 import static java.util.Arrays.asList;
 
+import static com.github.valfirst.slf4jtest.TestLoggerFactory.getTestLogger;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.stringContainsInOrder;
 import static org.junit.Assert.assertThat;
-import static uk.org.lidalia.slf4jtest.TestLoggerFactory.getTestLogger;
 
 import org.mule.runtime.core.internal.processor.LoggerMessageProcessor;
+import org.mule.runtime.module.artifact.api.classloader.MuleArtifactClassLoader;
 import org.mule.runtime.module.deployment.impl.internal.builder.ApplicationFileBuilder;
 import org.mule.runtime.module.deployment.internal.AbstractApplicationDeploymentTestCase;
 
 import java.util.List;
 
+import com.github.valfirst.slf4jtest.TestLogger;
 import io.qameta.allure.Description;
+import io.qameta.allure.Feature;
+import io.qameta.allure.Story;
 import org.junit.Test;
 import org.junit.runners.Parameterized;
-import uk.org.lidalia.slf4jtest.TestLogger;
 
+@Feature(INTEGRATIONS_TESTS)
+@Story(LOGGER)
```

Comment:
I'm not sure we are supposed to do this. The `LOGGER` story is part of the `CORE_COMPONENTS` feature. I don't know what happens if we mix these. Besides, I'm not sure what kind of feature `INTEGRATION_TESTS` would be. I'd prefer to use `CORE_COMPONENTS` as the feature here.